projects/iio_demo/src.mk

```diff
@@ -10,8 +10,7 @@ SRCS +=	$(NO-OS)/util/list.c \
 SRCS += $(DRIVERS)/adc/adc_demo/adc_demo.c \
 	$(DRIVERS)/adc/adc_demo/iio_adc_demo.c \
 	$(DRIVERS)/dac/dac_demo/iio_dac_demo.c \
-	$(DRIVERS)/dac/dac_demo/dac_demo.c \
-	$(DRIVERS)/api/irq.c
```

Comment:
I think irq is required by the aducm3029 platform driver / iio_app

Reply:
and by xilinx uart too